### ISSUE TITLE
V1.19 Game Crashes as soon as it laods

### ISSUE THREAD

**Issue by JKaiya** [finallion/The-Graveyard-FORGE-#41]
Noticed this is also a problem with Biomes O plenty and Ecologics
[crash-2022-06-16_21.51.17-client.txt](https://github.com/finallion/The-Graveyard-FORGE-/files/8924622/crash-2022-06-16_21.51.17-client.txt)

**Comment by finallion** [comment]
What makes you think this is related to the Graveyard?

**Comment by JKaiya** [comment]
> What makes you think this is related to the Graveyard?

I know this is a problem with this mod specifically as although I have other mods, I tested the mods one by one to know which mods do not work. I know Biomes O Plenty fixed this 2ish days ago from this reply post. Not sure about Ecologics though. When I added this mod to my mods folder, it crashed on the start up near the end like the other 2 mods I listed in the op. The Biomes O Plenty mod answered a issue post saying that it has something to do with the new forge version bugs, however they were able to fix the mod. I'm not a coder so honestly can't help you more than there.
